orchestra/static/orchestra/project/js/data_service.js

```diff
@@ -0,0 +1,90 @@
+var serviceModule =  angular.module('orchestra.project.services');
+
+serviceModule.factory('dataService', function() {
+  var dataService = new function() {
+    var that = this;
+    this._meta = {'tasks': {}};
```

Comment:
feels funny to underscore this, since private/public in javascript is handled by only returning an object with the public methods.